applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -10,19 +10,20 @@ menu "Bluetooth"
 
 config BLE_ACL_CONN_INTERVAL
 	int "Bluetooth LE ACL Connection Interval (x*1.25ms)"
-	default 80
+	default 32
```

Comment:
Was wondering if we can use 0x16 (20ms), since that will provide great improvement.
And also, if we low down the supervision timeout from 4 seconds to 1 second, then it is easier to trigger disconnection in poor radio enviornment, which might bring more stress to the system.

Reply:
Sure, we can do 20ms, I was just setting it as high as we could within the allowed range to save power. Let's discuss it on the technical sync today.